templates/vpc.yaml

```diff
@@ -502,6 +502,31 @@ Resources:
           Value: !Ref Project
         - Key: "OwnerEmail"
           Value: !Ref OwnerEmail
+  QuarantineSecurityGroup:
+    Type: 'AWS::EC2::SecurityGroup'
+    Properties:
+      GroupDescription: Security Group for bastians
+      VpcId:
+        Ref: "VPC"
+      SecurityGroupIngress:
+        - CidrIp: "10.0.0.0/8"
```

Comment:
shouldn't this be ingress from our VPN?

Reply:
it's shorter this way... but i updated it with the VPN range instead